README.rst

```diff
@@ -18,42 +17,40 @@ options, all from one common Sysl specification.
 
 The set of outputs is open-ended and allows for your own extensions. Sysl has been created with extensibility in mind and it will grow to support other representations over time.
 
-Installation
-------------
-If you are interested in trying out Sysl, you will need to install `Python 2.7 <https://www.python.org/downloads/>`_ .
+Prerequisites
+-------------
+If you are interested in trying out Sysl, you will need `Python 2.7 <https://www.python.org/downloads/>`_ .
 OSX users, we recommend installing Python 2.7 with `Homebrew <https://brew.sh/>`_  and adding it to the `PATH` in you `.bash_profile`::
 
   > brew install python
   > echo export PATH=/usr/local/opt/python/libexec/bin:\$PATH >> ~/.bash_profile
 
-You can then install Sysl with ::
-
-  > pip install sysl
+If you are behind a corporate proxy you might want to set up ``cntlm`` and a global ``pip.conf``
+file (see `official docs <https://pip.pypa.io/en/stable/user_guide/#config-file>`_ or `Stackoverflow <https://stackoverflow.com/a/46410817>`_).
```

Comment:
I'm undecided on this one. Do we want to address corporate firewall issues here? Most open source Python-related projects wouldn't do this, and it would probably start getting annoying if they did. Perhaps a compromise is to move environment-specific guidance into a separate linked doc.

Reply:
(Same goes for the existing material on macOS & brew.)

Reply:
Great idea. I've added an issue for that in order to get this PR merged if thats ok (#48 )

Reply:
Never mind, now that have to ask for another approval I might as well do it now.